pl_examples/domain_templates/computer_vision_fine_tuning.py

```diff
@@ -222,22 +222,23 @@ def forward(self, x):
         x = self.feature_extractor(x)
         x = x.squeeze(-1).squeeze(-1)
 
-        # 2. Classifier (returns logits):
+        # 2. Classifier (returns scores):
         x = self.fc(x)
 
-        return torch.sigmoid(x)
+        return x
 
     def loss(self, logits, labels):
         return self.loss_func(input=logits, target=labels)
 
     def training_step(self, batch, batch_idx):
         # 1. Forward pass:
         x, y = batch
-        y_logits = self.forward(x)
+        y_scores = self.forward(x)
+        y_logits = torch.sigmoid(y_scores)
```

Comment:
the variable names need to be switched now :) the sigmoid returns the probabilities and the forward returns the logits :) 
```suggestion
        y_logits = self.forward(x)
        y_scores = torch.sigmoid(y_logits)
```

Reply:
and the same in the other places :)

Reply:
Done